packages/react-router/docs/api/Route.md

```diff
@@ -178,3 +178,11 @@ When `true`, a `path` that has a trailing slash will only match a `location.path
 | `/one` | `/one` | yes |
 | `/one` | `/one/` | no |
 | `/one` | `/one/two` | no |
+
+## location: object
+
+A `<Route>` element tries to match its `path` to the current browser `location`.
+However, a [`location`](location.md) object with a different `pathname` can be passed for matching, instead of the current browser location.
```

Comment:
Although, one may point that since URLs have *path*s and not *pathname*s, removing `object` won't be a problem.